questions.md

```diff
@@ -36,7 +36,8 @@
 2. How would you have implemented the `getNextTierPromotionTime(Customer)` method?
 3. What do you think about the other parts of the class? How would you refactor?
     
-# Questions about 'API-endpoints.md'
+# Questions about 'API-endpoints.md' (use your IDE's support for folding code to present all endpoints in a single page)
```

Comment:
For the comment about using your IDE's support for folding code to make sense, this should be changed to 

> Questions about the `openapi.yaml`

Reply:
Done.